server/index.js

```diff
@@ -47,13 +54,55 @@ app.get('/songsByBandID/:id', async (req, res) => {
   }
 });
 
-app.post('/')
+app.post('/song', async (req, res) => {
+  // console.log('Request data: ', req.body);
+  try {
+    var song = await Song.create(req.body);
+    // console.log('Is there a song here?: ', song);
+    // console.log('Request body: ', req.body);
+    res.status(201).json({
+      status: 'Created new song!',
+      data: song
+    });
+  } catch (error) {
+    console.log('Error creating new song: ', error);
```

Comment:
should send back an error status code here

Reply:
Ok, I just did.